clustergroup/templates/subscriptions.yaml

```
@@ -54,6 +56,8 @@ spec:
   {{- end }}
   {{- if $.Values.global.options.useCSV }}
   startingCSV: {{ $subs.csv }}
+  {{- else if $subs.csv }}
+  startingCSV: {{ $subs.csv }}
```

Academically... I wonder if this would work (or be any clearer)

```suggestion
  {{- if coalesce $subs.csv $.Values.global.options.useCSV }}
  startingCSV:   {{ coalesce $subs.csv $.Values.global.options.useCSV }}
```

---

```suggestion
  {{- $useCSV := coalesce $subs.csv $.Values.global.options.useCSV }}
  {{- if $useCSV }}
  startingCSV: {{ $useCSV }}
```

---

As Lester correctly pointed out to me global.options.useCSV is a bool so we cannot really coalesce it with the per subscription CSV fiels. I'll go ahead and merge it, since it is what he and I tested